packages/exchange/src/app.module.ts

```diff
@@ -61,6 +62,15 @@ const getEnvFilePath = () => {
         DepositWatcherModule,
         WithdrawalProcessorModule
     ],
+    // exports: [
+    //     TypeOrmModule,
+    //     ScheduleModule,
+    //     MatchingEngineModule,
+    //     TradeModule,
+    //     OrderModule,
+    //     DemandModule,
+    //     OrderBookModule
+    // ],
```

Comment:
Should this be commented out?

Reply:
Yeah. The should be removed, as we are not importing them one-by-one by as a whole module.